src/rgw/rgw_sal.h

```diff
@@ -191,6 +191,9 @@ enum AttrsMod {
   ATTRSMOD_MERGE   = 2
 };
 
+static constexpr uint32_t FLAG_LOG_OP = 0x0001;
```

Comment:
the sheer number of locations this flags argument will be needed highlights why we should make it a flagset, rather than a bool :).  I'm not the czar of zipper interfaces, but I am comfortable with this design